.github/workflows/build-gateway-chart.yml

```diff
@@ -16,11 +16,22 @@ concurrency:
 
 env:
   REGISTRY_REPOSITORY: ghcr.io/grafbase/helm-charts
+  REGISTRY_USERNAME: grafbase
   CHART_PATH: gateway/helm
 
 jobs:
+  clone-repo:
+    name: Clone Repository
```

Comment:
why is this needed?

Reply:
This clones the current repository in order to be accessible to the commands we want to run

Reply:
We do it here already no? https://github.com/grafbase/grafbase/pull/2781/files#diff-e9fa7d2d7c793a00500d3f196767edba4a5b0124d747ddc714dd45b442243914L43